deployments/stages/1_libraries.ts

```diff
@@ -16,12 +17,13 @@ import {
 
 import { ERC20Wrapper } from '../../artifacts/ts/ERC20Wrapper';
 import { ExchangeIssuanceLibrary } from '../../artifacts/ts/ExchangeIssuanceLibrary';
-import { RebalancingHelperLibraryContract } from '../../types/generated/rebalancing_helper_library';
+import { IssuanceLibrary } from '../../artifacts/ts/IssuanceLibrary';
```

Comment:
Wanted to say that I renamed `IssuanceLibrary` to `CoreIssuanceLibrary` in my PR